src/slskd/Program.cs

```diff
@@ -154,6 +177,7 @@ public static class Program
 
         private static IConfigurationRoot Configuration { get; set; }
         private static OptionsAtStartup OptionsAtStartup { get; } = new OptionsAtStartup();
+        private static ILogger Log { get; set; } = new ConsoleWriteLineLogger();
```

Comment:
This is primarily an improvement for developers.  The logger isn't configured until later, but messages are sent to stdout early on.  By splitting the logger configuration out to `ConfigureGlobalLogger()` it is less clear that the logger shouldn't be used until after that is executed.

Adding this generic implementation as a default allows the logger to be used prior to configuration.

Also worth mentioning that a lot of this was also due to the fact that `Log` needed to become a module level property when the existing code was split out.